examples/sveltekit-pwa/src/app.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" href="/favicon.svg" type="image/svg+xml">
+    <link rel="apple-touch-icon" href="/pwa-192x192.png">
+    <link rel="mask-icon" href="/favicon.svg" color="#FFFFFF">
+    <meta name="msapplication-TileColor" content="#FFFFFF">
+    <meta name="theme-color" content="#ffffff">
```

Comment:
inconsistent spacing
```suggestion
		<link rel="icon" href="/favicon.svg" type="image/svg+xml">
		<link rel="apple-touch-icon" href="/pwa-192x192.png">
		<link rel="mask-icon" href="/favicon.svg" color="#FFFFFF">
		<meta name="msapplication-TileColor" content="#FFFFFF">
		<meta name="theme-color" content="#ffffff">
```